### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by LeonardCModoran**
Added more angularjs nice to have practices.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -324,6 +328,18 @@ module.factory('Service', function ($rootScope, $timeout, MyCustomDependency1, M
...
   ```
+  
+   or another good practice
+  
+   ```JavaScript
+  app.controller('MainCtrl', MainCtrl);
+
+  function MainCtrl () {
+    var main = this;
+    main.title = 'Some title'; //a clearer visual connection on how is defined on the view
```
Comment (mgechev) [inline README.md]: Would you move the comment above the line, i.e.:

```javascript
function MainCtrl () {
   var main = this;
   // a clearer visual connection on how is defined on the view
   main.title = 'Some title';
   ...
}
```

File: README.md [hunk 2]
```diff
@@ -249,11 +250,13 @@ Services | lowerCamelCase | dataFactory | others
 This will make your testing easier and in some cases prevent unexpected behaviour (for example, if you missed `$scope.$apply` in `setTimeout`).
 
 * Automate your workflow using tools like:
-    * [Yeoman](http://yeoman.io)
-    * [Gulp](http://gulpjs.com)
-    * [Grunt](http://gruntjs.com)
-    * [Bower](http://bower.io)
-
+	* [NPM](https://www.npmjs.com/)
```
Comment (mgechev) [inline README.md]: Would you change the indentation to spaces instead of tabs?

**Comment by mgechev** [comment]
Thanks for the PR! Would you address my comments and squash into a single commit?

**Comment by LeonardCModoran** [comment]
Ah, sorry about that, I don't know all the rules :D ... I will fix all of them. (was easier it it was plain html), do you know any good tutorial on github style guides? I've worked till now on svn repos, and private gitlab, and only recently github, so kind of new...

**Pull request closed without merging**